cpp/oneapi/dal/algo/subgraph_isomorphism/backend/cpu/graph.hpp

```diff
@@ -211,61 +193,68 @@ void graph<Cpu>::init_list_representation(const dal::preview::detail::topology<s
             std::int64_t edge_attr = 0;
             std::int64_t vertex_2 = t._cols[t._rows[i] + j];
 
-            graph_data_storage.plist_data->data[i][j] = vertex_2;
+            p_edges_list[i][j] = vertex_2;
             if (edge_attr >= 0 || has_edges_attribute) {
-                if (graph_data_storage.plist_data->edges_attribute[i] == nullptr) {
-                    graph_data_storage.plist_data->edges_attribute[i] =
-                        allocator_.allocate<std::int64_t>(degree);
+                if (p_edges_attribute[i] == nullptr) {
+                    p_edges_attribute[i] = allocator.allocate<std::int64_t>(degree);
```

Comment:
Looks like degree variable value depends on function argument ('t'). If so, you should also check that you have enough memory in p_edges_attribute[i] if it isn't nullptr.

Reply:
allocate throws an exception if memory allocation is failed. see [here](https://github.com/oneapi-src/oneDAL/blob/f13832bcfdb846b51c9746b4d5fff1cdcd2df076/cpp/oneapi/dal/algo/subgraph_isomorphism/backend/cpu/inner_alloc.hpp#L39)